challenges/2021_cloud_base_height/01_send_data_to_zarr.ipynb

```diff
@@ -0,0 +1,4235 @@
+{
```

Comment:
<p>It might be helpful to also frame the problem, with a section saying why do we need machine learning ready data, and what is wrong with the this data as we currently have that makes it not machine learning ready</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MetOffice/data_science_cop/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>